python/ray/util/annotations.py

```diff
@@ -46,16 +46,14 @@ def PublicAPI(*args, **kwargs):
         ...     return y
     """
     if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
-        return PublicAPI(stability="stable", api_group="others")(args[0])
+        return PublicAPI(stability="stable", api_group="Others")(args[0])
 
     if "stability" in kwargs:
         stability = kwargs["stability"]
         assert stability in ["stable", "beta", "alpha"], stability
-    elif kwargs:
```

Comment:
remove this condition since the key can also be `api_group` (in addition to `stability`)